jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/common/PacmanSdkConstants.java

```diff
@@ -259,8 +259,13 @@ public interface PacmanSdkConstants {
 
     /** The status exempted. */
     String STATUS_EXEMPTED = "exempted";

```

Comment:
This can be removed as the final status is always Exempt

Reply:
@AnandPaladin instead of fixing the UI layer, you need to make sure the problem at root is also setting the "Exempt". 

You can then leave the hack in place for legacy logs, however new logs should have the proper status set!

Reply:
@plyubich Whenever a violation is exempted, the status is stored as `exempted` in ES. But in UI we are showing that as `Exempt`.

But in the Audit log, we use `Exempt` for user action and `Exempted` for system logs. As we are using `Closed` should we also use `Exempted` everywhere?